libs/template/templates/dbt-sql/template/{{.project_name}}/resources/{{.project_name}}.job.yml.tmpl

```diff
@@ -46,3 +53,17 @@ resources:
               spark.databricks.cluster.profile: singleNode
             custom_tags:
               ResourceClass: SingleNode
+          {{- end }}
+
+      {{- if $with_serverless }}
+
+      environments:
+        - environment_key: default
+
+          # Full documentation of this spec can be found at:
+          # https://docs.databricks.com/api/workspace/jobs/create#environments-spec
```

Comment:
This comment actually doesn't seem very useful. The only other property where we do this is the `schedule`, where it is important that customers understand the implications.

(I also meant to remove this from the `lakeflow-pipelines` template but I see it snuck back in there. And would like to remove it from `default-python`.)